Sources/SecureXPC/Client/XPCClient.swift

```diff
@@ -268,8 +290,8 @@ public class XPCClient {
         fatalError("Abstract Property")
     }
 
-    /// Creates and returns a connection for the service represented by this client.
-    internal func createConnection() -> xpc_connection_t {
+    /// Creates and returns a connection for the service represented by this client or throws an error if it's unable to do so.
```

Comment:
Unnecessary addition to the documentation IMO.

"or throws an error if it's unable to do so" is what `throws` already means

Reply:
Fair enough